src/authorization/service/search.service.ts

```diff
@@ -3,6 +3,7 @@ import { Brackets, ObjectLiteral, SelectQueryBuilder } from 'typeorm';
 import { SearchEntity } from '../../constants/search.entity.enum';
 import {
   GroupSearchInput,
+  OperationType,
```

Comment:
This seems to be an unused import and can be removed.

Reply:
Have removed and therefore marking as resolved.